research/src/pages/popup/popup.research.explainer.mdx

```diff
@@ -75,6 +75,32 @@ This proposal was discussed on [Issue 455](https://github.com/openui/open-ui/iss
 
 There have been **many** discussions and resolutions at OpenUI of various aspects of this proposal, and those are listed in the [Design Decisions](#design-decisions) section.
 
+## Pop-Up vs. Dialog
+
+A natural question that [commonly arises](https://github.com/openui/open-ui/issues/581) is: what's the difference between a "pop-up" and a "dialog"? There are two ways to interpret this question:
+ 1. What are the difference between these general UX patterns and words?
+ 2. What are the technical implementation differences between `<div popup>` and `<dialog>`?
+
+In both cases, an important distinction needs to be made between **modal** and **non-modal** (or **modeless**) dialogs. With a **modal** dialog, the rest of the page (outside the dialog) is rendered **`inert`**, so that only the contents of the dialog are interactable. Importantly, a pop-up is **non-modal**. Almost by definition, a pop-up is "ephemeral" and goes away (via light dismiss) if the user changes their attention to something else, by clicking or tabbing to something else. For these reasons, if the dialog in question needs to be **modal**, then the `<dialog>` element is the way to go.
+
+Having said that, dialogs can also be **non-modal**, and pop-ups can be set to not light-dismiss (via **`popup=manual`**). There is a significant area of overlap between the two Web features. Some use cases that lie in this area of overlap include:
+
+1. "Toasts" or asynchronous notifications, which stay onscreen until dismissed manually or via a timer.
+2. Persistent UI, such as teaching-UI, that needs to stay on screen while the user interacts with the page.
+3. Custom components that need to "manually" control pop-up behavior.
+
+Given these use cases, it's important to call out the technical differences between a non-modal `<dialog>` and a `<div popup=manual>`:
+
+- A `<div popup=manual>` is **in the top layer**, so it draws on top of other content. The same is not true for a non-modal `<dialog>`. This is likely the most impactful difference, as it tends to be difficult to ensure that a non-modal `<dialog>` is painted on top of other page content.
+- A `<dialog>` element always has **`role=dialog`**, while the `popup` attribute can be applied to the **most-semantically-relevant HTML element**, including the `<dialog>` element itself: `<dialog popup>`.
+- The pop-up API comes with some **"nice to have's"**:
+    - pop-ups are easy to animate both the show and hide transitions. JS is required to animate `dialog.close()`.
+    - pop-ups fire both a "show" and a "hide" event. A `<dialog>` only fires `cancel` and `close`, but no event is fired when it is shown.
+    - pop-ups work with the invoking attributes (e.g. `popuptoggletarget`) to declaratively show/hide them. JS is required to show/close a `<dialog>`.
+
+For the above reasons, it seems clear that `<div popup=manual>` is superior to non-modal `<dialog>` in most cases, and should be preferred.
```

Comment:
this is the only line i don't completely agree with - specifically because of the statement `div popup=manual` is superior' and that it should be preferred. 

I'd rather this state something like if someone needs a modeless dialog to have popup behavior, that they use `<dialog popup / popup=manual>`.  if they need a popup that is not meant to be exposed as a dialog, then use `<div popup=...>` and as necessary add the appropriate `role`.

Reply:
Yep, great point. Take a look at the new version and LMK if that works for you.

Reply:
lgtm.  thank you!